compiler/rustc_target/src/spec/nto_qnx_base.rs

```diff
@@ -0,0 +1,18 @@
+use crate::spec::{cvs, TargetOptions};
+
+pub fn opts() -> TargetOptions {
```

Comment:
Full relro is set in the backend gcc compiler.  Does it make sense to have that option set here for clarity?

Reply:
It is now set to `RelroLevel::Full`. It was disabled before. Thanks!